app/javascript/packages/document-capture/higher-order/with-background-encrypted-upload.jsx

```diff
@@ -31,12 +31,15 @@ export function blobToDataView(blob) {
  */
 export async function encrypt(key, iv, value) {
   const data =
-    typeof value === 'string' ? new TextEncoder().encode(value) : await blobToDataView(value);
+    typeof value === 'string' ? new TextEncoder().encode(value) : await blobToArrayBuffer(value);
 
   return window.crypto.subtle.encrypt(
     /** @type {AesGcmParams} */ ({
       name: 'AES-GCM',
       iv,
+      // Normally, it would not be expected to assign this value, since the property is optional and
+      // the default is 128. However, if not specified, Internet Explorer will throw an error.
+      tagLength: 128,
```

Comment:
should we add some sort of "expect to be called with" expectation to enforce this 128 sticks around?

Reply:
🤔  Yeah, that sounds like a good idea. It might even make sense to have it a more global expectation within tests, i.e. "if `crypto.subtle.encrypt` is called with `{ name: 'AES-GCM' }`, make sure `tagLength` is included.

On the other hand, we ideally shouldn't need to be considering this, since I'd expect that the crypto polyfill / shim would be handling this browser inconsistency for us (see also https://github.com/vibornoff/webcrypto-shim/pull/44).

Reply:
Ok, happy to trust your gut on this. The comment is probably enough to cover it, so I'd expect that change to jump out in a diff if somebody moved the code and didn't bring `tagLength` along

Reply:
Added test helper in e11661030112c10391890ebcacf8eacb951d31e1.